examples/repository_deployment_policy/main.tf

```diff
@@ -0,0 +1,19 @@
+resource "github_repository" "terraformed" {
+  name        = "terraformed"
+  description = "A repository created by Terraform"
+}
+
+resource "github_repository_environment" "terraformed" {
+  repository = github_repository.terraformed.name
+  environment = "terraformed"  
+  deployment_branch_policy {
+    protected_branches          = false
+    custom_branch_policies = true
+  }
```

Comment:
```suggestion
  repository = github_repository.terraformed.name
  environment = "terraformed"  
  deployment_branch_policy {
    protected_branches     = false
    custom_branch_policies = true
  }
```

Reply:
Hey @jjgriff93 thank you for your review. This folder was used by me to test out the functionality and I ended up deleting it, so I'm gonna mark the formatting comment as resolved.